docs/modules/ROOT/pages/install.adoc

```diff
@@ -24,34 +23,47 @@ The following files are available in the tar file:
 | Apache Cassandra 4.x | Luna Streaming | agent-c4-luna-<version>-all.jar
 | DSE 6.8.16+ | Apache Pulsar | agent-dse4-pulsar-<version>-all.jar
 | DSE 6.8.16+ | Luna Streaming | agent-dse4-luna-<version>-all.jar
```

Comment:
Please double check with @eolivelli about the versions. I think the versions with "-luna" in the naming will be removed